test/test_ai_player.py

```diff
@@ -53,6 +53,16 @@ def testAIPlayerTriesToWinIfAbleToTopRow(self):
         expected_result = 1
         self.assertEqual(result, expected_result, msg='\nRetrieved:\n{0} \nExpected:\n{1}'.format(result, expected_result))
 
+    def testAIPlayerTakesWinInMiddleOfTopRow(self):
```

Comment:
This may be a bit of a nitpick.  The AIPlayer was earlier in the life of the project, and I don't think this is used anymore.  I'd also wonder about the likelihood that someone changing code here would break this.

Reply:
Also, I believe the `AIPlayer` is no longer used.  Normally, you would want to delete this entire class if it is no longer used in the program.